vm/interpreter/src/lib.rs

```diff
@@ -40,6 +40,11 @@ where
         .get_actor(addr)?
         .ok_or_else(|| anyhow::anyhow!("Failed to retrieve actor: {}", addr))?;
 
+    // If there _is_ an f4 address, return it as "key" address
+    if let Some(address) = act.delegated_address {
```

Comment:
We don't need to test for `StateTree::V5` here, since we don't want to support previous network version, right?

Reply:
It's a new field. Previous versions should have it set to `None`. Checking the state tree version is redundant. Feel free to correct me if you think otherwise.

Reply:
No, I think it's good like this.